component-library/src/app/pages/spinner-documentation/spinner-documentation.component.scss

```diff
@@ -0,0 +1,15 @@
+@use '../../../../../core-library/util/device' as device;
+@use "../../../scss/page";
+@use "../../../scss/grid";
+@use '@ircc-ca/ds-sdc-core/typography/typography';
+
+@include page.template;
+
```

Comment:
Configurations section should maintain 2 column layout in mobile view. Michael just made a similar fix in the button documentation for reference.

Reply:
Thanks Bobby, updated Configurations section in mobile view for Spinner, Banner, DatePicker, IconBtn and Input components